src/Calculator/Calculator.java

```diff
@@ -0,0 +1,48 @@
+package Calculator;
+
+public class Calculator {
+    private char[] defaultOperators = {'+', '-', 'x', '%', '/', '!'};
+    private char[] charArray;
+
+    public int calculate(String expression) {
+        if (expression.isEmpty() || expression.equals("")) {
+            return 0;
+        }
+        charArray = expression.toCharArray();
+        int result = Integer.parseInt(String.valueOf(charArray[0]));
+        for (int i = 1; i < charArray.length; i = i + 2) {
+            if ((i + 1) != charArray.length) {
+                result = operation(result, charArray[i], Integer.parseInt(String.valueOf(charArray[i + 1])));
+            } else result = operation(result, charArray[i], 0);
+        }
```

Comment:
Looks exactly like https://github.com/Almantask/Java-Homework/pull/21/files 
Same issues as there...
Did you help each other out?

Reply:
No, I did all by myself. Changed function, so it can calculate with many digits.